PD-282 symbol update toggles to new palette

audroue opened the pull request:
- Changed a lot colors to align with standardized color palelette and limited use of exception colors (there are now like 3 exception blues)
- Removed press state
- Added more layer styles
- Added focus and hover borders

audroue committed:
In Progress work

- Updated cover
- Resized some art-boards
- Removed downpress toggle state gradient
- Changing colors

audroue committed:
Merge branch 'master' into PD-282-Symbol-Update-Toggles-to-New-Palette

audroue committed:
PD-272 Symbol Update Toggles to New Palette

- Removed press states
- Made a lot of color changes
- Added blue border to focus dark states (off)
- Now limited number of exception colors (to I think 3 different blues)

DesignerDave commented:
Hey Rob!

Looked through the changes and I have a few notes:

First, I don't think the active blue is well enough aligned with the palette. The more vibrant exception blue is more cool of a color than blues of similar values in our range of blues. We may want to adjust the hue of the active color to make it more seamlessly work with our palette. Below (top: base, middle: new toggle, bottom: our focus blue (likely to be `light 1` in the future)):
<img width="343" alt="Screen Shot 2019-09-24 at 12 09 09 PM" src="https://user-images.githubusercontent.com/25213299/65529559-586cea80-dec4-11e9-9ad5-ce28a710db6a.png">

---

Second, I know this was also previously the case, but we should probably constrain the height and width of the toggle symbols. The height and width aren't variable in the React component, and should probably remain consistent across implementations of the toggle.
<img width="516" alt="Screen Shot 2019-09-24 at 12 09 34 PM" src="https://user-images.githubusercontent.com/25213299/65529565-5d319e80-dec4-11e9-9d57-225fad521329.png">

---

Third, the border on the dark toggle is an in-between color between the active state of the toggle, and the focus ring, and causes the toggle to appear blurry. We may want to darken the border when focused+active to increase its contrast against the focus ring, or explore other solutions for this issue.
<img width="361" alt="Screen Shot 2019-09-24 at 12 17 15 PM" src="https://user-images.githubusercontent.com/25213299/65530063-43dd2200-dec5-11e9-8a08-12e798a9732e.png">

<img width="385" alt="Screen Shot 2019-09-24 at 12 16 28 PM" src="https://user-images.githubusercontent.com/25213299/65529994-28721700-dec5-11e9-920e-d84093e24c81.png">

audroue committed:
Figured out way to lock sizing and position of toggle

Figured out way to lock sizing and position of toggle

audroue committed:
Fixing sizing of toggles

audroue committed:
Updates based on feedback

- constrained size of all toggles
- changed color of light mode active state toggle to better align with palette (it's now a color between "base" and "light 1"
- added dark border to mitigate "blur effect" on active dark mode active & focused state toggle

DesignerDave reviewed (changes requested):
It looks like the sizing issues with the toggles is still persistent.

![Screen Shot 2019-11-12 at 2 36 53 PM](https://user-images.githubusercontent.com/25213299/68704170-e460d180-0559-11ea-8f5b-684f2c135ebb.png)

Consumers shouldn't be able to change the toggle's size (except through size variants).

audroue commented:
Hey Dave! In the file I'm looking at, I'm not seeing that resize issue. Might have to ask you in person, when you have a sec.
